TreeTracker/UI/Views/Authentication/Selfie/SelfieViewController.swift

```diff
@@ -64,8 +64,8 @@ private extension SelfieViewController {
     func displayImagePicker() {
         let imagePicker = UIImagePickerController()
         imagePicker.delegate = self
-        imagePicker.sourceType = .camera
-        imagePicker.cameraDevice = .front
+//        imagePicker.sourceType = .camera
```

Comment:
Get why you did this for dev purposes (the main branch now has a photo library option for debug builds to resolve this) but they need to be uncommented before merging.

Reply:
Ditto